diagnostic.md

```diff
@@ -36,13 +39,16 @@ Now, we want to get a **single donut** from the server. Write the `curl` request
 you'd use to retrieve a **single donut**, using whatever ID you'd like.
 
 ```sh
-# your answer here
+# curl --include --request GET http://www.example.com/donut
```

Comment:
Almost.  `donut` should be `donuts/1` (or whatever number you want).

`curl --include --request GET http://www.example.com/donuts/1`